### PR TITLE
This is a working branch.

### PR DESCRIPTION
**Pull request by Harcourt1**
This code works for local and server hosting. Dev is already at this branch but this is what the server is running and what you should run also.

### CODE REVIEW

File: config/dbconfig.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@ FEEDBACK_DATABASE feedback
 FEEDBACK_TABLEPREFIX 
 
 ## Username/Login used to access the database.
-FEEDBACK_LOGIN FO13
+FEEDBACK_LOGIN deathclaw
```
Comment (AndrewL97) [inline config/dbconfig.txt]: Good work publicly posting the sql username/passwords

File: config/admins.txt [hunk 2]
```diff
@@ -11,8 +11,9 @@ LiquidRecluse = Elder
 Verhulst = Elder
 ComradeSully = Elder
 Harcourt = President
```
Comment (AndrewL97) [inline config/admins.txt]: You're already in the main branch as host, but can you check with dmast if this should be the case?

File: config/admin_ranks.txt [hunk 3]
```diff
@@ -37,4 +37,5 @@ Elder   		= +EVERYTHING
 
 President		= +EVERYTHING
 
-Scribe			= +DEBUG +VAREDIT +SERVER +SPAWN
\ No newline at end of file
+Scribe			= +DEBUG +VAREDIT +SERVER +SPAWN
+Linux Manager		= +DEBUG +VAREDIT +SERVER +SPAWN +BAN
```
Comment (AndrewL97) [inline config/admin_ranks.txt]: Same with admins.txt comment
why is this a new role?

File: config/config.txt [hunk 4]
```diff
@@ -17,7 +17,7 @@ ROUND_END_COUNTDOWN 90
 ADMIN_LEGACY_SYSTEM
 
 ## Add a # infront of this if you want to use the SQL based banning system. The legacy systems use the files in the data folder. You need to set up your database to use the SQL based system.
-#BAN_LEGACY_SYSTEM
+BAN_LEGACY_SYSTEM
```
Comment (AndrewL97) [inline config/config.txt]: Why aren't we using sql db for bans?

**Comment by AndrewL97** [comment]
Review's up @WrongEnd 

I dont see what changes are necessary here. Especially the SQL config which should live on our server. @Harcourt1 has just posted the root sql details publicly, which means if somebody gets access they have our player's IP addresses, ckeys, etc.

**Comment by WrongEnd** [comment]
@AndrewL97 I'm mostly ad libbing when it comes to code, which is why I had you review before I merged. If he doesn't justify the changes and fix the issues you brought up, I'll close the PR and delete the branch.

**Comment by WrongEnd** [comment]
Will give harcourt another 12 hours, then close the PR and delete the branch.

**Comment by WrongEnd** [comment]
More than enough time given. Closing.

**Pull request closed without merging**